vnext/Scripts/IntegrationTests.ps1

```diff
@@ -1,97 +1,94 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-param(
-	[switch] $NoSetup,
-	[switch] $Run,
-	[switch] $Cleanup,
+#
+# IntegrationTests.ps1
+#
+param (
+	[switch] $NoServers,
+
 	[ValidateSet('x64', 'x86')]
 	[string] $Platform = 'x64',
+
 	[ValidateSet('Debug', 'Release')]
 	[string] $Configuration = 'Debug',
+
 	[string[]] $Tests,
-	$Assembly = "$PSScriptRoot\..\target\$Platform\$Configuration\React.Windows.Desktop.IntegrationTests\React.Windows.Desktop.IntegrationTests.dll",
-	$VsTest = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow\vstest.console.exe",
+
+	[switch] $Preload,
+
 	[UInt32] $Delay = 10,
+
+	[ValidateScript({Test-Path $_})]
+	[string[]] $Assemblies =
+	(
+		"$PSScriptRoot\..\target\$Platform\$Configuration\" +
+		"React.Windows.Desktop.IntegrationTests\React.Windows.Desktop.IntegrationTests.dll"
+	),
+
+	[switch] $NoRun,
+
 	[switch] $List,
-	[switch] $Preload,
-	[string] $ReactNativeHome = "$($PSScriptRoot | Split-Path)\node_modules\react-native"
-)
 
-function Cleanup() {
-	# Expected netstat header:
-	# P1 P2     P3                     P4                     P5              P6
-	#    Proto  Local Address          Foreign Address        State           PID
-	try {
-		$process = (netstat -ano | Select-String ':8081')[0] | ConvertFrom-String
-		(Get-Process -Id $process.p6).Kill()
-	}
-	catch {
-		Write-Host 'Metro bundler process not found.'
-	}
+	[ValidateScript({Test-Path $_})]
+	[string] $VsTest =	"${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\Enterprise\" +
```

Comment:
>2019\Enterprise [](start = 70, length = 15)

What about folks that are still on VS 2017, or that use a SKU other than Enterprise? Do we express a VS2019-Enterprise requirement somewhere (readme, setup scripts etc)?

Reply:
It is not a requirement, but if they choose to use VS2017, they will have to provide their own path to the VSTest console.